homeassistant/components/sia/.translations/en.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I'm not sure this file is up to date

Reply:
I thought I deleted that one, will check.